doc/build-macOS-Sierra-Qt.txt

```diff
@@ -0,0 +1,153 @@
+Build ionx-Qt on macOS 10.12+ Sierra
+
+confirmed to work with
+miniupnpc 2.0
+boost 1.64.0_1
+qt 5.8.0_2
+openssl 1.0.2k
+berkeley-db4 4.8.30
+gmp 6.1.2
+
+Install Xcode https://developer.apple.com/download
+Install the associated Command Line Tools to your Xcode version https://developer.apple.com/download/more
+
+Launch Terminal
+Install brew from http://brew.sh
+
+Brew some recipes!
+brew install autoconf automake berkeley-db4 gmp libpng libtool miniupnpc openssl qrencode qt5
+brew install boost --c++11 --without-single --without-static
+
+Manually link your openssl dependency paths to the brew cellar (recommend /usr/local/opt/openssl/lib and /usr/local/opt/openssl/include):
```

Comment:
Sorry I should clarify, I thought Homebrew automatically linked `/usr/local/opt/openssl` when you `brew install openssl`